src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -1051,6 +1078,24 @@
         ]
       }
     },
+    "toolkitNamespace":{
+      "type": "generated",
+      "generator": "switch",
+      "replaces": "$toolkitNamespace$",
+      "parameters": {
+        "evaluator": "C++",
+        "cases": [
+          {
+            "condition": "(useToolkit)",
+            "value": "xmlns:utu=\"using:Uno.Toolkit.UI\""
```

Comment:
Can we maybe put in some spaces and a line return `\n` so that way we don't have a weird empty line when we're not using Toolkit.

Reply:
yep good call